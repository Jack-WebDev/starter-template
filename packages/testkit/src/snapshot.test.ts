import { describe, expect, test } from "vitest";

describe("snapshot test", () => {
	test("basic object", () => {
		const user = { id: 1, name: "Jack" };
		expect(user).toMatchSnapshot();
	});
});
