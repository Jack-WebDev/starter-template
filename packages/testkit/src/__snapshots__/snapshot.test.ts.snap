// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshot test > basic object 1`] = `
{
  "id": 1,
  "name": "Jack",
}
`;
